tests/test_initrd.py

```diff
@@ -40,13 +40,20 @@ def passphrase() -> Iterator[Path]:
 
 @pytest.fixture(scope="module")
 def initrd(config: Image.Config) -> Iterator[Image]:
-    with Image(
-        config,
-        options=[
-            "--directory", "",
-            "--include=mkosi-initrd/",
-        ],
-    ) as initrd:
+    with (
+        ci_group(
+            f"Initrd image {config.distribution}/{config.release} using {config.tools_tree_distribution} tools tree"
+            if config.tools_tree_distribution else
+            f"Initrd image {config.distribution}/{config.release} (no tools tree)"
```

Comment:
I don't see the point of logging about whether we use a tools tree or not. Just add a summary dump before the build instead if you want that info logged.